src/08_comprehensions.py

```diff
@@ -10,32 +10,44 @@
 
```

Comment:
One of the coolest features of python, it may be odd to use at first compared to JavaScript, try it as much as you can